src/_guides/language/sound-problems.md

```diff
@@ -122,6 +122,58 @@ Otherwise, use `dynamic` in situations where you cannot use a single type:
 var width = canvasOrImg.width;
 {% endprettify %}
 
+#### Example 2
```

Comment:
With the second example, we need to clarify the scope of the first and second examples and their fixes. Maybe:

```
#### Example 1: <description>
##### Fix: <existing description>
#### Example 2: <description>
##### Fix: <description>
```